### DISCUSSION TITLE
deepin-editor: 希望有预览列

### DISCUSSION THREAD

**Discussion by Decodetalkers** [linuxdeepin/developer-center#2850] [Features Request & Ideas | 特性请求 & 头脑风暴]
像kwrite vscode的侧边
如果能支持最基本的cpp的联想就更好了，希望看齐kwrite

**Comment by Decodetalkers** [comment]
另外全屏上面有标签页，完全不沉浸

**Comment by guoshaoyu** [comment]
> 像kwrite vscode的侧边
> 如果能支持最基本的cpp的联想就更好了，希望看齐kwrite

感谢您的建议，我们会合理考虑此功能。
